devices/nous.js

```diff
@@ -57,7 +57,7 @@ module.exports = [
             e.temperature(),
             e.humidity(),
             e.battery(),
-            exposes.enum('temperature_unit_convert', ea.STATE_SET, ['celsius', 'fahrenheit']).withDescription('Current display unit'),
+            exposes.enum('temperature_unit_convert', ea.STATE_SET, ['°C', '°F']).withDescription('Current display unit'),
```

Comment:
Please change the to/fromZigbee converters to use `celsius` and `fahrenheit` instead.

Reply:
Update fromZigbee.js #3888
Update toZigbee.js #3889